executor/modes/local/runner.py

```
@@ -0,0 +1,407 @@
+            task_title=task_data.get("task_title", ""),
+            subtask_title=task_data.get("subtask_title", ""),
+        )
+
+        # Store progress reporter in task_data for agent use
+        task_data["_local_progress_reporter"] = progress_reporter
+
+        # Create and initialize agent
+        self.current_agent = ClaudeCodeAgent(task_data)
+
+        # Override report_progress to use WebSocket
+        original_report_progress = self.current_agent.report_progress
```

Ruff (0.14.14)

Local variable `original_report_progress` is assigned to but never used

Remove assignment to unused variable `original_report_progress`

(F841)